tests/test_node_licenses.py

```diff
@@ -15,10 +15,10 @@
 from tests.base import OsfTestCase
 from tests.utils import assert_logs, assert_not_logs
 from website import settings
-from website.project.licenses import (NodeLicense, ensure_licenses,
-                                      serialize_node_license,
-                                      serialize_node_license_record)
-from website.project.model import NodeLog, NodeStateError
+from osf.models.licenses import NodeLicense, serialize_node_license_record, serialize_node_license
+from website.project.licenses import ensure_licenses
```

Comment:
This would also be good as a data migration, but it might be out of scope for this PR.